test/test_jit_cuda_fuser.py

```diff
@@ -3154,7 +3154,7 @@ def t_bias(x: torch.Tensor, w: torch.Tensor, bias: torch.Tensor):
             jit_o = jitted_bias(inp, weight, bias)
 
         graph = jitted_bias.graph_for(inp)
-        self.assertGraphContainsExactly(graph, FUSION_GROUP, 0)
+        self.assertGraphContains(graph, FUSION_GROUP, True)
```

Comment:
hmmm, are we having multiple fusion here because of the new view op?

Reply:
I am seeing a `view + bias` fusion here.